YouVsVirus/Assets/Scripts/Components/nonSpawnableSpace.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+// The nonSpawnableSpace class serves the purpose to
+// defines regions on the map that are non-spawnable.
```

Comment:
defines --> define

Reply:
You can actually directly suggest such small changes here in the github UI, like this:
```suggestion
// define regions on the map that are non-spawnable.
```
It is then very easy for my to just click a button and commit them.

Reply:
would that no be to easy, dear sir?

Reply:
to --> too

Reply:
![grafik](https://user-images.githubusercontent.com/7125317/81843167-81beb180-954d-11ea-9035-83bf3017ee52.png)

Reply:
no --> not